stac_fastapi/extensions/stac_fastapi/extensions/core/aggregation/request.py

```diff
@@ -0,0 +1,31 @@
+"""Request model for the Aggregation extension."""
+
+from typing import Optional
+
+import attr
+from pydantic import BaseModel
+
+from stac_fastapi.types.search import APIRequest
+
+
+@attr.s
```

Comment:
can `filter` be included in this? I'm not sure how the extensions interact.

Reply:
Yeah, I have tested with filter included on an implementation and it works great.

I am also not clear on how extension interact. For instance, could `collections`, `datetime`, `bbox`, and `intersects` be excluded here since they are part of the core search? I'm inclined to keep so the request class correlates directly to the extension spec.

Reply:
yeah, i think you should extend BaseSearchGetRequest and BaseSearchPostRequest and just add the `aggregations` param. You also pickup `limit` that's ignored, but that's find for a datamodel like this since there's a lot less duplication

Reply:
I have changed it to be build off base Search and Filter. This makes the implementation dependent on the Filter extension. But I don't think that is an issue.

Reply:
@jamesfisher-gis sorry I'm just realizing this now but I don't think we should do this, at least I don't see in the `aggregate` extension why the `filter` attributes should be enabled with the `aggregations` ones. 

I think it will be up to the implementers to add the `filter` extension and also handle them in the client.

Reply:
Hey. That makes sense.

I have a couple other small bug fixes for aggregation. I will submit a PR today that removed the Filter extension dependency.

Reply:
maybe wait because I will submit a PR that takes care of #713 soon

Reply:
@jamesfisher-gis in fact, go ahead because my PR is a no-go in fact 😓